ConfigSpace/hyperparameters.pyx

```diff
@@ -2556,7 +2556,14 @@ cdef class CategoricalHyperparameter(Hyperparameter):
             Probability density values of the input vector
         """
         probs = np.array(self.probabilities)
+        vector = np.atleast_1d(vector)
+        nan = np.isnan(vector)
+        if np.any(nan):
+            vector[nan] = 0
         res = np.array(probs[vector.astype(int)])
+        if np.any(nan):
+            res[nan] = 0
+            vector[nan] = np.nan
```

Comment:
This is not used any more and appears to be not necessary.

Reply:
Removed line 2566